entry-generation/godot-entry-generator/src/main/kotlin/godot/entrygenerator/exceptions/WrongAnnotationUsageException.kt

```diff
@@ -0,0 +1,14 @@
+package godot.entrygenerator.exceptions
+
+import org.jetbrains.kotlin.descriptors.PropertyDescriptor
+import org.jetbrains.kotlin.descriptors.annotations.AnnotationDescriptor
+import org.jetbrains.kotlin.resolve.descriptorUtil.fqNameSafe
+
+class WrongAnnotationUsageException(
```

Comment:
I guess this will be temporary until we moved to using `MessageCollector`?

Reply:
Yes. But something like this should also be used for the message collector so the error messages are the same.